packages/probing-frontend/src/pages/MonitoringDetailPage/components/MonitoringEserviceProbing.tsx

```diff
@@ -0,0 +1,118 @@
+import type { ProbingEservice } from '@/api/monitoring/monitoring.models'
+import { Alert, Box, Chip, Grid, Typography } from '@mui/material'
+import { ButtonNaked } from '@pagopa/mui-italia'
+import { useTranslation } from 'react-i18next'
+import RefreshIcon from '@mui/icons-material/Refresh'
+import { formatDateString } from '@/utils/date.utils'
+import type { TranslationKeys } from './MonitoringInformationContainer'
+import { MonitoringInformationContainer } from './MonitoringInformationContainer'
+
+export const MonitoringEserviceProbing = ({
+  eservicesProbingDetail,
+  refetch,
+}: {
+  eservicesProbingDetail: ProbingEservice | undefined
+  refetch: () => Promise<void>
+}) => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const getChipColor = (value: string) => {
+    switch (value) {
+      case 'ONLINE':
+      case 'true':
+        return 'success'
+      case 'OFFLINE':
+      case 'false':
```

Comment:
Why are we handling `true` and `false` cases here? Where they come from?

Reply:
@Carminepo2  function `const getChipColor: (value: string) => "success" | "error" | "warning" | undefined` is used to return a string for both `probingEnabled` case (which returns a boolean)  and `state` case (which returns a `ProbingStatusType`)

Reply:
`getChipColor` now is

```
const getChipColor: (value: ProbingStatusType | 'true' | 'false') => "error" | "success" | "warning" | undefined
```

after commit [ce0addf](https://github.com/pagopa/interop-be-probing/pull/8/commits/ce0addf211bfd7b91610bc860d83dedea6c0bd29)

Reply:
I would probably split the two logic. Handle the `probingEnabled` chip color with a ternary and transforming the function `getChip` in `getProbingStateChipColor`, that handles only the color of that one specified chip